libmscore/fraction.h

```diff
@@ -233,7 +233,8 @@ class Fraction {
             // MScore::division     - ticks per quarter note
             // MScore::division * 4 - ticks per whole note
             // result: rounded (MScore::division * 4 * _numerator * 1.0 / _denominator) value
-            const auto result = (static_cast<int_least64_t>(_numerator) * MScore::division * 4 + (_denominator/2)) / _denominator;
+            const int sgn = (_numerator < 0) ? -1 : 1;
```

Comment:
I don't know what the performance benefits would be, by the way. Maybe it is quicker to create a const and access it twice than it is to evaluate a comparison between two ints twice?

Reply:
It is hard to say it a priori. Is it called so often so it is critical for a performance? If so, it is definitely better to measure whether there is a measurable performance difference between these two (and other) options.